aiidalab_widgets_base/utils/loaders.py

```diff
@@ -0,0 +1,38 @@
+from importlib.resources import Package, files
+
+from IPython.display import Javascript, display
+
+
+def load_css_stylesheet(package: Package, filename: str = ""):
+    """Load a CSS stylesheet from a package and inject it into the DOM.
+
+    Parameters
+    ----------
+    `package` : `Package`
+        The package where the CSS file is located.
+    `filename` : `str`, optional
+        The name of the CSS file to load.
+        If not provided, all CSS files in the package will be loaded.
+    """
+    root = files(package)
+
+    filenames = []
+    if filename:
+        filenames.append(filename)
+    else:
+        filenames.extend(
+            path.name
+            for path in root.iterdir()
+            if path.is_file() and path.name.endswith(".css")
+        )
+
+    for filename in filenames:
+        stylesheet = (root / filename).read_text()
```

Comment:
nit: This for loop var shadows the function parameter. Not a huge deal but I am surprised that ruff does not complain about it.

```suggestion
    for f in filenames:
        stylesheet = (root / f).read_text()
```

Reply:
Indeed surprising. Done!